easybuild/easyconfigs/q/QIIME/QIIME-1.9.1.eb

```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Conda'
+
+name = 'QIIME'
+version = '1.9.1'
+
+homepage = 'http://qiime.org/'
+description = """QIIME is an open-source bioinformatics pipeline for performing microbiome analysis
+ from raw DNA sequencing data."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+requirements = "%(namelower)s=%(version)s matplotlib=1.4.3 mock nose"
+channels = ['bioconda']
```

Comment:
@pescobar please also include `conda-forge` in the `channels` list? better safe than sorry, I guess...